backend/src/test/java/gov/cdc/usds/simplereport/api/model/TestEventExportTest.java

```diff
@@ -145,4 +147,22 @@ void determinesFirstTestFromTestEvent_hasPriorTest_null() {
 
     assertEquals("UNK", sut.getFirstTest());
   }
+
+  @Test
+  void specimenCollectionSubtractsDeviceOffset() {
+    Organization o = _dataFactory.createValidOrg();
+    Facility f = _dataFactory.createValidFacility(o);
+    Person p = _dataFactory.createFullPerson(o);
+    TestEvent te = _dataFactory.createTestEvent(p, f, TestResult.NEGATIVE, null);
+
+    var testLength = f.getDefaultDeviceType().getTestLength();
+    var specimenTime =
+        Date.from(te.getDateTested().toInstant().plus(Duration.ofMinutes(testLength)));
+    TestEventExport sut = new TestEventExport(te);
+
+    assertEquals(
+        TestEventExport.dateToHealthCareString(
```

Comment:
This test uses the same logic as the implementation for the expected, meaning any flaw in the production code will also change the test expectation, thus leading to test to pass. I recommend hard-coding the expectation. and reverting the static methods to private again.

Reply:
Resolved in 3c43bf37